client/my-sites/stats/annual-site-stats/index.js

```diff
@@ -0,0 +1,120 @@
+					<div className="annual-site-stats__stat-title">{ translate( 'year' ) }</div>
+					<div className="annual-site-stats__stat-figure is-large">{ data.year }</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'total posts' ) }</div>
+					<div className="annual-site-stats__stat-figure is-large">
+						{ numberFormat( data.total_posts ) }
+					</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'total comments' ) }</div>
+					<div className="annual-site-stats__stat-figure">
+						{ numberFormat( data.total_comments ) }
+					</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">
+						{ translate( 'avg comments per post' ) }
+					</div>
+					<div className="annual-site-stats__stat-figure">
+						{ numberFormat( data.avg_comments ) }
+					</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'total likes' ) }</div>
+					<div className="annual-site-stats__stat-figure">{ numberFormat( data.total_likes ) }</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'avg likes per post' ) }</div>
+					<div className="annual-site-stats__stat-figure">{ numberFormat( data.avg_likes ) }</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'total words' ) }</div>
+					<div className="annual-site-stats__stat-figure">{ numberFormat( data.total_words ) }</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'avg words per post' ) }</div>
+					<div className="annual-site-stats__stat-figure">{ numberFormat( data.avg_words ) }</div>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { years, translate, moment } = this.props;
+		const now = moment();
+		const currentYear = now.format( 'YYYY' );
+		let previousYear = null;
+		if ( now.month() === 0 ) {
+			previousYear = now.subtract( 1, 'months' ).format( 'YYYY' );
+		}
+		const currentYearData = years && find( years, y => y.year === currentYear );
+		const previousYearData = previousYear && years && find( years, y => y.year === previousYear );
+		const isLoading = ! years;
+		const isError = ! isLoading && years.errors;
+		const noData = ! isLoading && ! isError && ! ( currentYearData || previousYearData );
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Annual Site Stats', { args: [ currentYear ] } ) } />
+				<Card className="stats-module">
+					<StatsModulePlaceholder isLoading={ isLoading } />
+					{ isError && <ErrorPanel message={ translate( 'Oops, something went wrong' ) } /> }
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Oops! Something went wrong.' )` *ES Score: 21*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/annual-site-stats)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).